decNumber/include/decNumber/decSingle.h

```diff
@@ -38,9 +38,9 @@
   #define DECSINGLE_Ehigh (DECSINGLE_Emax + DECSINGLE_Bias - (DECSINGLE_Pmax-1))
 
   /* Required includes                                                */
-  #include "decContext.h"
-  #include "decQuad.h"
-  #include "decDouble.h"
+  #include "include/decNumber/decContext.h"
+  #include "include/decNumber/decQuad.h"
+  #include "include/decNumber/decDouble.h"

```

Comment:
change these 3 to use the `#include <decNumber/dec*.h>` form?

Reply:
Yes. I thought I fixed these, apparently I missed a couple.